TreeList: Add onNodesInitialized Example Link (#8928)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxTreeList/1 Configuration/onNodesInitialized.md

The root node.

---

> Use **onNodesInitialized** to modify [node fields](/api-reference/10%20UI%20Components/dxTreeList/4%20Node '/Documentation/ApiReference/UI_Components/dxTreeList/Node/'). Call [forEachNode(callback)](/api-reference/10%20UI%20Components/dxTreeList/3%20Methods/forEachNode(callback).md '/Documentation/ApiReference/UI_Components/dxTreeList/Methods/#forEachNodecallback') in this function to iterate over nodes and apply changes.

Reviewer: ```suggestion
Use **onNodesInitialized** to modify [node fields](/api-reference/10%20UI%20Components/dxTreeList/4%20Node '/Documentation/ApiReference/UI_Components/dxTreeList/Node/'). In the event handler, call [forEachNode(callback)](/api-reference/10%20UI%20Components/dxTreeList/3%20Methods/forEachNode(callback).md '/Documentation/ApiReference/UI_Components/dxTreeList/Methods/#forEachNodecallback') to iterate through nodes and apply changes.
```

> You can also implement custom iteration logic using the **root** parameter. To traverse the node tree, access each node's **children** field starting from **root** in a recursive function. The following example configures custom iteration logic in **onNodesInitialized** to display a summary for each TreeList branch:

Reviewer: ```suggestion
You can also implement custom iteration logic with the **root** (`e.root`) parameter. To traverse the node tree, start from the **root** and recursively access each node's **children** field. The following example configures custom iteration logic in **onNodesInitialized** to display a summary for each TreeList branch:
```

> You can also implement custom iteration logic using the **root** parameter. To traverse the node tree, access each node's **children** field starting from **root** in a recursive function. The following example configures custom iteration logic in **onNodesInitialized** to display a summary for each TreeList branch:

Author: We can avoid replacing "using" with "with" as follows:
```suggestion
You can also use the **e**.**root** field to implement custom iteration logic. To traverse the node tree, start from **root** and recursively access each node's **children** field. The following example configures custom iteration logic in **onNodesInitialized** to display a summary for each TreeList branch:
```
